packages/console/src/assets/docs/single-sign-on/AzureAD/README.mdx

```diff
@@ -71,27 +74,29 @@ You may manage the user attributes mapping logic either on the Azure AD side or
 
 1. Map the AzureAD user attributes to Logto user attributes at Logto side.
 
-<center>
-  <img src={defaultAttributesMapping} alt="Default Attributes Mapping" />
-</center>
+   Visit the `Attributes & Claims` section of your Azure AD SSO application.
 
-Visit the `Attributes & Claims` tab of your Azure AD SSO application.
+   Copy the following attribute names (with namespace prefix) and paste them into the corresponding fields in Logto.
 
-Copy the following attribute names (with namespace prefix) from the `Attributes & Claims` section of your Azure AD SSO application and paste them into the corresponding fields in Logto.
+   - `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/email`
 
-- `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/email`
+   - `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name` (Recommendation: update this attribute value map to `user.displayname` for better user experience)
 
-- `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name` (Recommend: update the attribute value map to `user.displayname` for better user experience)
+<center>
```

Comment:
nit: can we use markdown image syntax and adjust css globally to set small images to center?

Reply:
Tried markdown syntax, can't read the assets files path properly some how. Haven't taken a deep look at it yet. May be later.